go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/stretchr/testify v1.9.0
 	github.com/vmware-tanzu/net-operator-api v0.0.0-20240326163340-1f32d6bf7f9d
 	github.com/vmware-tanzu/nsx-operator/pkg/apis v0.1.0
+	github.com/vmware-tanzu/vm-operator v1.8.6
```

Comment:
Question: do we want to avoid this? I did see we tried to in test: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/test/framework/vmoperator/vmoperator.go#L788

Reply:
Note: this comes from importing `github.com/vmware-tanzu/vm-operator/pkg/conditions`